### PR TITLE
Events: Add 3rd party software distribution event

### PR DESCRIPTION
**Pull request by jalehman**

### CODE REVIEW

File: content/events/2021-04-15-developer-call-software-distribution.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+youtube = "XwICC6Its1E"
++++
+
+Some exciting work has been going on behind the scenes to provide the foundation for 3rd party software distribution. In this talk Phil (~wicdev-wisryt) and Raghu (~sovmep-ripsum) will discuss the work they've been doing to clay to provide "virtual desks"—an integral part of the infrastructure needed to have safe, generic software distribution.
+
+Raghu is a software engineer at Hudson River Trading who is perhaps best known for his work on ucal, the Urbit calendar application. The work he's doing with Phil on virtual desks is being done through our Scholarships program, in which community developers pair up with Tlon infrastructure engineers to work on infrastructure projects. We'll spend some time at the beginning of the call to introduce Raghu and hear about his experience working on the Urbit kernel.
+
+As usual, there'll be an opportunity to jump on the call with us afterward to ask questions and hang out.
+
+Original mailing list discussion of 3rd party software distribution: https://groups.google.com/a/urbit.org
```
Comment (tylershuster) [inline content/events/2021-04-15-developer-call-software-distribution.md]: Suggest to link these

**Comment by jalehman** [comment]
Any idea what's going on here @tylershuster @urcades ?

Screen Shot 2021-04-14 at 1 29 42 PM

**Comment by tylershuster** [comment]
lol Ed must have targeted `#rsvp` in his last CSS change, affecting all RSVP links

**Comment by jalehman** [comment]
> lol Ed must have targeted `#rsvp` in his last CSS change, affecting all RSVP links

https://github.com/urbit/urbit.org/pull/596/files#diff-bbddd2c8891c4a8f8990d003243420bfc9d370cfd8cf99fce017033866cc4d12R696-R711

**Comment by urcades** [comment]
@jalehman ugh,

this is likely one of the results of my ugly hacking-together to get the landing page looking the way I desired — I highly recommend changing the template used for the event to `template = "events/event.html"` in order to use the previously-built styling for 'blog-looking' landing pages. It should work the way you expect it to, I think.

**Comment by tylershuster** [comment]
Well actually™ the `#rsvp` CSS is in the main CSS file, and events use `events/event.html` by default, which has its own [`#rsvp` element](https://github.com/urbit/urbit.org/blob/26fb2379442e73ee48b6cd5551379ea606bd0cc6/templates/events/event.html#L65) so it gets applied everywhere

**Comment by jalehman** [comment]
Fixed with the least ugly solution I could think of in a short period of time.

**Pull request merged**